app/index.js

```diff
@@ -139,7 +139,7 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       backgroundColor: toElectronBackgroundColor(cfg.backgroundColor || '#000'),
       // we want to go frameless on windows and linux
       frame: process.platform === 'darwin',
-      transparent: true,
+      transparent: process.platform === 'darwin',
```

Comment:
transparent doesn't work on Linux unless you run with --disable-gpu and a laundry-list of other caveats :(